### ISSUE TITLE
Version for android?

### ISSUE THREAD

**Issue by GTD4life** [stefankueng/CryptSync#18]
I love this program!! Since mobile security is  rising in importance an android version would be wonderful. Just a wish :-).

**Comment by CryptoStar2000** [comment]
Very good idea! An Android version would be great. Is there any way to organize support for this?

**Comment by github-dinoop** [comment]
Yes. Making an encrypted backup of selected folders by connecting a cloud storage would be great.

**Comment by Tau-Neutrino** [comment]
I really like cryptsync. I use it mainly to sync my personal documents and pictures encrypted to cloud storage!

A version for android would be absolutely perfect! Any plans for it? Or does somebody know an alternative working like cryptsync available for android?

**Comment by ismgr** [comment]
RCX is similar...
[https://f-droid.org/en/packages/io.github.x0b.rcx/](url)

**Comment by Dr-Nuke** [comment]
what a wonderful piece of software. If I could sync the encrypted files in my dropbox/ onedrive to a local folder on my android smartphone, that would be outstanding. upvote!
